find_package(gazebo) broken on Windows

The PR https://github.com/osrf/gazebo/pull/2864 , in particular https://github.com/osrf/gazebo/pull/2864/files#diff-0838b3a1790a01bd4c5b10e6789d5b7309df76869413ea9e166ee36815653b70R110, broke the calls to `find_package(gazebo)` on Windows. The error that you get as soon as you call  `find_package(gazebo)` is:
~~~
CMake Error at C:/Users/STraversaro/Miniforge3/envs/robsup/Library/lib/cmake/gazebo/gazebo-config.cmake:146 (message):
  Library 'gazebo' in package GAZEBO is not installed properly
Call Stack (most recent call first):
  cmake/BuildGazeboYARPPlugins.cmake:31 (find_package)
  build-conda3/install/share/YCM/modules/FindOrBuildPackage.cmake:215 (include)
  cmake/RobotologySuperbuildLogic.cmake:12 (find_or_build_package)
  CMakeLists.txt:45 (include)
~~~
This is because the line in https://github.com/osrf/gazebo/blob/72f40624d4ef33166c8cb864390d6a1769044b14/CMakeLists.txt#L438 should be modified in Windows to read `libgazebo`  instead of `gazebo` .

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -434,8 +434,19 @@ else (build_errors)
   # Order is important, if A depends on B, please add B after A.
   # The list should have at the very end the libraries
   # without internal interdependencies
-  set(PKG_LIBRARIES
-    gazebo
+  
+  # The gazebo library on Windows is called libgazebo to avoid 
+  # conflicts with the gazebo executable
+  if(NOT WIN32)
+    set(PKG_LIBRARIES
+      gazebo
+    )
+  else()
+    set(PKG_LIBRARIES
+      libgazebo
+    )
+  endif()  
+  set(PKG_LIBRARIES ${PKG_LIBRARIES}
     gazebo_client
     gazebo_gui
     gazebo_sensors

